vxyowsup/whatsapp.py

```diff
@@ -107,9 +115,19 @@ def _kill():
         self.stack.execDetached(_stop)
         self.stack.execDetached(_kill)
 
+    def send_to_stack(self, text, toAddress):
+        # encapsulate text and address?
+        # rather have it interact with stack only?
+        self.stack.execDetached(self.echo_layer.send, text, toAddress)
```

Comment:
`execDetached` only accepts a function with no arguments (it's not as cool as many other callback handlers). This means we'll need to do something like:
```python
def send_to_stack(self, text, toAddress):
    def send():
       self.echo_layer.send(text, toAddress)
    self.stack.execDetached(send)
```
or, if we want to be cool and use Python's `functools` module:
```python
def send_to_stack(self, text, toAddress):
    self.stack.execDetached(functools.partial(self.echo_layer.send, text, toAddress))
```
which does the same thing.